examples/mcp/transmcp.jl

```diff
@@ -55,17 +55,15 @@ using Test
     @complementarity(m, supply, w)
     @complementarity(m, fxdemand, p)
 
-    PATHSolver.options(convergence_tolerance=1e-8, output=:yes, time_limit=3600)
 
-
-    status = solveMCP(m)
+    status = solveMCP!(m; convergence_tolerance=1e-8, output="yes", time_limit=3600)
 
     @show result_value.(x)
     @show result_value.(w)
     @show result_value.(p)
 
     @show status
-    @test status == :Solved
+    @test status == PATHSolver.MCP_Solved
```

Comment:
Changing the return status is also breaking. Is there a need to?

Reply:
You are right. No good reason to break existing users' codes.